test/framework/integration_framework/iroha_instance.hpp

```diff
@@ -89,9 +92,9 @@ namespace integration_framework {
 
     // config area
     const std::string block_store_dir_ = "/tmp/block_store";
-    const std::string redis_host_ = getRedisHost();
-    const size_t redis_port_ = getRedisPort();
-    const std::string pg_conn_ = getPostgreCreds();
+    const std::string redis_host_ = getRedisHostOrDefault();
+    const size_t redis_port_ = getRedisPortOrDefault();
+    const std::string pg_conn_ = getPostgreCredsOrDefault();
```

Comment:
You don't need many methods. Look how configs are implemented in `trunk/cliutils`: https://github.com/hyperledger/iroha/blob/trunk/cliutils/test/integration/pipeline/tx_pipeline_integration_test.cpp#L104

You can at least reuse this method:
https://github.com/hyperledger/iroha/blob/trunk/cliutils/libs/util/string.hpp#L73

And make better configs:
https://github.com/hyperledger/iroha/blob/trunk/cliutils/test/integration/pipeline/tx_pipeline_integration_test.cpp#L42-L46

Reply:
Separate task was created for this issue.